deploy/examples/monitoring/rbac.yaml

```diff
@@ -27,9 +32,9 @@ roleRef:
   kind: Role
   name: rook-ceph-monitor
 subjects:
-- kind: ServiceAccount
-  name: rook-ceph-system
-  namespace: rook-ceph
+  - kind: ServiceAccount
```

Comment:
Can you check the indentation again? Seems to be a bit off as an extra space is added in all the changes.

Reply:
Actually a go indent is added or been updated,

So if you touch any of the file that is old format will change